spec-0008/index.md

```diff
@@ -0,0 +1,225 @@
+# SPEC-8: Securing the Release Process
+
```

Comment:
```suggestion
```
This needs to go away to render correctly as a table at the beginning of the document and in the website I think. At least that's how it's formatted in other SPECS.